packages/@lwc/ssr-compiler/src/compile-template/shared.ts

```diff
@@ -172,3 +177,11 @@ export function getChildAttrsOrProps(
 
     return b.objectExpression(objectAttrsOrProps);
 }
+
+/**
+ * Determine if the provided node is of type Literal
+ * @param node
+ */
+export function isLiteral(node: IrLiteral | IrExpression | IrComplexExpression): node is IrLiteral {
+    return node.type === 'Literal';
+}
```

Comment:
`is.literal` does the same thing. We should be able to just use that instead.

Reply:
`is.literal` works on estree nodes, this helper is for template nodes.

Reply:
That's definitely a point of confusion for me currently 😅 more helpers like this will be nice.

Reply:
My bad. We have a bunch of these in `@lwc/template-compiler` but they are not exported.

Reply:
Would it be worth me exporting them and consume those instead as we are already pulling in that package, or?